### PR TITLE
upstream v1.3.8

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: overlay.js [hunk 1]
```diff
@@ -0,0 +1,586 @@
...
+            "   }",
+            "   #popup > .image {",
+            "       float: left;",
+            "       margin: 16px 16px 0 16px;",
+            "   }",
+            "   #popup > .description {",
+            "       text-align: left;",
+            "   }",
+            "   #popup > #popupInstall {",
+            "       mergin: 0;",
```
Comment (cvan) [inline overlay.js:293]: typo. should be `margin`.

Comment (cvan) [inline overlay.js:293]: /cc @Maksims

**Comment by cvan** [comment]
r+

File: overlay.js (hunk 1 above)
Comment (Maksims) [inline overlay.js:293]: Thanks for that one :)

**Pull request merged**